inst/shiny/server.R

```diff
@@ -5,6 +5,7 @@ options(shiny.autoreload = TRUE)
 
 internetConnection <- suppressWarnings(Biobase::testBioCConnection())
 source("partials.R", local = TRUE) # creates several smaller UI components
+source("server_partials/server_01_data.R", local = TRUE) # functions for Data section
```

Comment:
Excuse me, but where is this file please?

I got the following error when trying to start the app after merging new commits. 
```
Warning in file(filename, "r", encoding = encoding) :
  cannot open file 'server_partials/server_01_data.R': No such file or directory
Warning: Error in file: cannot open the connection
  52: file
  51: source
Error in file(filename, "r", encoding = encoding) : 
  cannot open the connection
```

Reply:
Sorry, that was a mistake; it should have been commented out.  I'll fix it and push again!

Reply:
Addressed in PR #307